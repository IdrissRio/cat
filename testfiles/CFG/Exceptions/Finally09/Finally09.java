public class Finally09 {
  void m2() {
    try {
      int a = 0;
      if (a > 0)
        return;
    } finally {
      int b;
    }
    int c;
  }
}