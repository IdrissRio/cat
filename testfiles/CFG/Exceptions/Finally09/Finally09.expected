NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [IntegerLiteral[49]]
NodeName: IntegerLiteral[49] Successors: [VariableDeclarator[46]]
NodeName: VariableDeclarator[46] Successors: [VarAccess[52]]
NodeName: VarAccess[52] Successors: [IntegerLiteral[53]]
NodeName: IntegerLiteral[53] Successors: [GTExpr[51]]
NodeName: GTExpr[51] Successors: [UncheckedExceptions[55], UncheckedExceptions[83]]
NodeName: UncheckedExceptions[55] Successors: [VariableDeclarator[66]]
NodeName: VariableDeclarator[66] Successors: [ReturnStmt[54], Exit[36]]
NodeName: ReturnStmt[54] Successors: [Exit[36]]
NodeName: Exit[36] Successors: []
NodeName: UncheckedExceptions[83] Successors: [VariableDeclarator[80]]
NodeName: VariableDeclarator[80] Successors: [VariableDeclarator[89], Exit[36]]
NodeName: VariableDeclarator[89] Successors: [Exit[36]]

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[36] Predecessor: [ReturnStmt[54], VariableDeclarator[66], VariableDeclarator[80], VariableDeclarator[89]]
NodeName: ReturnStmt[54] Predecessor: [VariableDeclarator[66]]
NodeName: VariableDeclarator[66] Predecessor: [UncheckedExceptions[55]]
NodeName: UncheckedExceptions[55] Predecessor: [GTExpr[51]]
NodeName: GTExpr[51] Predecessor: [IntegerLiteral[53]]
NodeName: IntegerLiteral[53] Predecessor: [VarAccess[52]]
NodeName: VarAccess[52] Predecessor: [VariableDeclarator[46]]
NodeName: VariableDeclarator[46] Predecessor: [IntegerLiteral[49]]
NodeName: IntegerLiteral[49] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: VariableDeclarator[80] Predecessor: [UncheckedExceptions[83]]
NodeName: UncheckedExceptions[83] Predecessor: [GTExpr[51]]
NodeName: VariableDeclarator[89] Predecessor: [VariableDeclarator[80]]
